pkg/api/userapi/roles_test.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/elastic/cloud-sdk-go/pkg/multierror"
 )
 
+const platformViewerRole = "ece_platform_viewer"
```

Comment:
I think this constant should go live where it was as all the available user roles are in one place 🤔

Reply:
I wanted to do that, but since it isn't used anywhere but the tests, I had to move it to the test definition itself since it is otherwise considered unused.